modules/scsi_mock/scsi_mock_module.c

```diff
@@ -0,0 +1,138 @@
+#include <linux/device.h>
+#include <linux/init.h>
+#include <linux/kernel.h>
+#include <linux/module.h>
+#include <linux/scatterlist.h>
+
+#include <scsi/scsi.h>
+#include <scsi/scsi_host.h>
+
+#define NAME "SCSI2NVMe Mock"
```

Comment:
Prefer consts over macros. https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros (unless this is not possible in the module code)